Add support EGLImage-based texture

makotosato-at opened the pull request:
Hello.

I needed to add EGLImage-based texture support to flutter-embedded-linux to improve video_player playback performance.
So I added support.

Relevant issue: sony/flutter-elinux-plugins#64

HidenoriMatsubayashi commented:
Thanks for sending the PR.

Can you please update AUTHORS too and sign a DCO (Developer Certificate of Origin) in git comment?

Also, it's unclear whether we can contribute this software to flutter/engine, but do you agree to delegate all rights related to this PR to Sony?

HidenoriMatsubayashi commented:
It looks good to me, but please fix code format check error.

makotosato-at commented:
Thank you for your review.
I fixed code format check error and updated AUTHORS, singed a DCO.

> Also, it's unclear whether we can contribute this software to flutter/engine, but do you agree to delegate all rights related to this PR to Sony?

I agree to delegate all rights related to this PR to Sony.

HidenoriMatsubayashi reviewed (approved):
LGTM. Thanks a lot.
